composer.json

```diff
@@ -29,7 +29,8 @@
   "minimum-stability": "dev",
   "prefer-stable":     true,
   "require":     {
-    "dreamfactory/df-core": "~0.25"
+    "php": "^8.0",
```

Comment:
@daniilly @alexanderko Title says PHP 8.1 but you're requiring 8.0.
Please update to match the title

Reply:
Hello @anas-srikou 

During migration we were testing DF features on PHP 8.1. So so title. On the other hand changes made in this PR intruduce function added in PHP 8.0 and no special features of PHP 8.1. 

We relaxed PHP version requirement for 2 reasosons
* We don't use any features introduced in PHP 8.1 here